flake.nix

```diff
@@ -26,56 +26,41 @@
     poetry2nix = {
       url = "github:nix-community/poetry2nix";
     };
+
     darwin = {
       url = "github:lnl7/nix-darwin";
       inputs.nixpkgs.follows = "nixpkgs";
     };
+
     firefox-darwin.url = "github:bandithedoge/nixpkgs-firefox-darwin";
   };
   outputs = inputs:
-  let
-    lib = inputs.snowfall-lib.mkLib {
-      inherit inputs;
-      src = ./.;
-    };
-    wsl-modules = with inputs; [
-      nixos-wsl.nixosModules.wsl
-      nixos-vscode-server.nixosModules.default
-    ];
-  in
-    lib.mkFlake {
+    let
+      #TODO: rework this https://nix.dev/anti-patterns/language#with-attrset-expression
+      wsl-modules = with inputs; [
+        nixos-wsl.nixosModules.wsl
+        nixos-vscode-server.nixosModules.default
+      ];
+    in
+    inputs.snowfall-lib.mkFlake {
       inherit inputs;
-      lib = inputs.nixpkgs.lib;
-      overlay-package-namespace = "arichtman";
       src = ./.;
-      channels-config.allowUnfree = false; #TODO: remove if I'm really done with VSCode
-      #TODO: rework this https://nix.dev/anti-patterns/language#with-attrset-expression
+
+      package-namespace = "arichtman";
+
+      #TODO: remove if I'm really done with VSCode
+      channels-config.allowUnfree = false;
```

Comment:
(you probably want to keep this here, there's a lot of unfree packages that are useful)